enderio-machines/src/main/java/com/enderio/machines/data/recipe/enchanter/EnchanterRecipeResult.java

```diff
@@ -0,0 +1,53 @@
+package com.enderio.machines.data.recipe.enchanter;
+
+import javax.annotation.Nullable;
+
+import com.enderio.machines.EIOMachines;
+import com.enderio.machines.common.recipe.EnchanterRecipe;
+import com.google.gson.JsonObject;
+
+import net.minecraft.data.recipes.FinishedRecipe;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+
+public class EnchanterRecipeResult implements FinishedRecipe{
```

Comment:
Make ENchanterRecipeResult Generic, so it can be used by other recipes as well

Reply:
Not sure what you mean

Reply:
forget it, had an idea, but doesn't really work I think

Reply:
forget it, had an idea, but doesn't really work I think